plugins/woocommerce/includes/emails/class-wc-email.php

```diff
@@ -256,6 +256,25 @@ public function __construct() {
 
 		add_action( 'phpmailer_init', array( $this, 'handle_multipart' ) );
 		add_action( 'woocommerce_update_options_email_' . $this->id, array( $this, 'process_admin_options' ) );
+		add_action( 'woocommerce_email_footer', array( $this, 'mobile_messaging' ), 9 ); // Run before the default email footer.
```

Comment:
I'm not sure why there's `priority 9`. Should `mobile_messaging` be executed before `woocommerce_email_footer` (as your comment says) or I didn't understand it?

<img width="333" alt="image" src="https://user-images.githubusercontent.com/5845095/187238499-c219e0ef-d2dc-4d9f-941c-6dd31cf4bcf6.png">

Reply:
Yeah, lower priority number means it will happen sooner. The default priority is `10`. We want this message to be output before the footer.